agent/profile

```diff
@@ -6,6 +6,13 @@ function _pbench_pathins() {
     fi
 }
 
+_pbench_pathins "/usr/local/sbin"
+_pbench_pathins "/usr/local/bin"
+_pbench_pathins "/usr/sbin"
+_pbench_pathins "/usr/bin"
+_pbench_pathins "/root/bin"
+_pbench_pathins "/sbin"
```

Comment:
These last two I am not sure of now that I see them.  I think it opens an unnecessary set of problems.

Reply:
Yes I did some tests with pbench-agent commands and `/sbin` doesn't seem to be required in the path (but I wasn't fully sure and its possible that I missed something that actually need this directory in the path so I added it thinking we dont want to come across an issue where something is missing from the system PATH)